install.sh

```diff
@@ -904,7 +904,7 @@ TasksMax=infinity
 TimeoutStartSec=0
 Restart=always
 RestartSec=5s
-ExecStartPre=/bin/sh -xc '! /usr/bin/systemctl is-enabled --quiet nm-cloud-setup.service'
+ExecStartPre=/bin/sh -xc '! /usr/bin/systemctl is-enabled --quiet nm-cloud-setup.service' 2>/dev/null
```

Comment:
I think this should go inside the single quotes? You want to redirect stderr from `systemctl`, not `/bin/sh`, right?
```suggestion
ExecStartPre=/bin/sh -xc '! /usr/bin/systemctl is-enabled --quiet nm-cloud-setup.service' 2>/dev/null
```

Reply:
The result is the same but probably your suggestion is less confusing :)